README.md

```diff
@@ -126,6 +127,49 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.raw_semian_options`
+that takes a `semian_identifier` like `http_127_0_0_1_80` or `http_github_com_80`,
+and returns a `Hash` with keys as follows, perhaps in an initialization file.
+
+```ruby
+SEMIAN_PARAMETERS = { tickets: 1,
+                      success_threshold: 1,
+                      error_threshold: 3,
+                      error_timeout: 10 }
+Semian::NetHTTP.raw_semian_options = proc do |semian_identifier|
+  # Let's make it only active for github.com
+  if semian_identifier == "http_github_com_80"
```

Comment:
The "not including port" thing will break a few tests for sure.

Also, going on that callback for the identifier thing, the adapter would need two callbacks, one for the uri -> identifier, and one for identifier -> parameters? We can't just merge them since the identifier is definitely needed for the name. That's quite a bit of user-involved customizing going on, best to have some default as well then.